core/src/main/resources/openapi-specs/eth2/signing/paths/high_watermark.yaml

```diff
@@ -0,0 +1,24 @@
+get:
+  tags:
+    - 'High Watermark'
+  summary: 'The High Watermark epoch and slot applicable to all validators'
+  description: 'Returns the uint64 epoch and slot of the high watermark. Signing of attestations or blocks are prevented at or beyond this high watermark'
```

Comment:
```suggestion
  description: 'Returns the uint64 epoch and slot of the high watermark. Signing of attestations or blocks are prevented at and beyond this high watermark'
```

Reply:
I think "or" is more correct here because current object having its signature prevents will either by at or beyond the high watermark. Also used this terminology in test names

Reply:
no worries, I think it's worth clarifying in which situation each could happen. When I first read this I thought this was a mistake.

Reply:
Could reverse it: "Signing of attestations or blocks are only allow when they are lower than this high watermark"?

Reply:
much clearer